### ISSUE TITLE
Tableau de bord évolution mensuelle des notes de satisfaction

### ISSUE THREAD

**Issue by AnthonyBrunelli** [DISIC/wikidemarches#931]
ven. 03/07, 11:01 de G. Bayard
Bonjour Anca,
 
J’aime vraiment beaucoup l’évolution qui permet d’accéder aux évolutions mensuelles de note. Y aurait-il un moyen d’accéder à cette donnée de manière plus concentrée, sous forme de tableau de bord, histoire d’accéder à une information plus synthétique et exploitable  ? genre un tableau qui permettre de visualiser les évolutions les plus fortes facilement
 
Bien cordialement,

Réponse d'Anca: [...] Ludovic a ajouté quelques outils de statistiques privés (uniquement à usage interne pour des admins), il a peut-être quelque-chose qui peut répondre à ce besoin.

En fonction du besoin, je me demande si au niveau d'outil de stats d'accès (Matomo(Piwik)) il y aura pas des outils similaires. Ce que Matomo ne peut pas savoir c'est la valeur du vote (Bien, Moyen, Mauvais) mais il peut tout à fait avoir les informations sur le trafic, qui sont proportionnelles aux nombres de votes et donc  permettraient d'avoir une vision comparative d'une même démarche dans le temps mais aussi des démarches entre elles).

**Comment by lucaa** [comment]
Un outil pour les administrateurs a été ajouté dans le commit 274988e4fb06e5cd3a6398ffe1695dfb9f159818 (21 Oct) pour écrire un tableau avec les évolutions des démarches dans une page.

**Comment by georges-bayard** [comment]
Il faut en reparler. J'ajoute en PJ le résultat auquel nous voulons in fine arriver que tu n'as peut être pas vu @lucaa . Nous sommes partis de l'extraction que tu m'avais envoyé et que nous avons mouliné. peut être y a t'il moyen d'améliorer plus en amont l'extraction elle même.
[EvolutionsDemarchesSept2020-top250-100plus.xlsx](https://github.com/DISIC/wikidemarches/files/5617439/EvolutionsDemarchesSept2020-top250-100plus.xlsx)

**Comment by lucaa** [comment]
> Il faut en reparler. J'ajoute en PJ le résultat auquel nous voulons in fine arriver que tu n'as peut être pas vu @lucaa . Nous sommes partis de l'extraction que tu m'avais envoyé et que nous avons mouliné. peut être y a t'il moyen d'améliorer plus en amont l'extraction elle même.

Oui, j'avais vu qu'il y avait eu un travail par dessus, mais pas encore analysé en détail pour transformer cela en amélioration détaillé et précise. Effectivement, il va falloir reparler, c'est pour cela que je garde ce ticket encore ouvert, pour tracer ce reste de travail à faire.

**Comment by AnthonyBrunelli** [comment]
Pertinence à évaluer @jrenauxdinum et @ugodtn

**Comment by jrenauxdinum** [comment]
Cela nous permettrait d'être encore plus réactif dans l'analyse de la satisfaction effectivement ... A voir ce que cela représente en termes de dev :)
